sdk/python/pvsite_datamodel/write/forecast.py

```diff
@@ -63,11 +62,9 @@ def insert_forecast_values(
 
         # For each target time:
         for target_time in target_times:
-            # Convert the target_time string to a datetime object
```

Comment:
could you make it optional?

Reply:
Is there a specific case where dates will be ISO strings?
I know that in `pv-site-production` the data is already a datetime object.
Unless ISO string is a very common scenario, I would tend to keep it simple and let the caller do the conversion. This way we could have tighter typing.

Reply:
I'm on Simon's side here: since it's a function in a python library that can only get used in other python scripts, enforcing it to be a stdlib datetime object seems to make better use of the language. Were it an API or something that could be called by multiple languages/frameworks, then an ISO-formatted string (or even a UNIX timestamp string) would be the more sensible choice.

Reply:
yea, go with what @simlmx says then